boards/trellis_m4/src/lib.rs

```diff
@@ -78,6 +87,132 @@ define_pins!(
     pin dotstar_di = b3,
 );
 
+impl Pins {
```

Comment:
Maybe move this section into the pins module.

Reply:
I tried to move the `Pins` struct into the `pins` module, but for whatever reason the `define_pins!` macro isn't accessible there (and you can't `impl` a type that lives in a different module).

Reply:
Specifically:

```
error: cannot find macro `define_pins!` in this scope
 --> src/pins.rs:6:1
  |
6 | define_pins!(
  | ^^^^^^^^^^^
```